Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -9179,6 +9179,13 @@ void WebPageProxy::drawFooterForPrinting(WebFrameProxy& frame, FloatRect&& rect)
     m_uiClient->drawFooter(*this, frame, WTFMove(rect));
 }
 
+void WebPageProxy::drawPageBorderForPrinting(WebFrameProxy& frame, WebCore::FloatSize&& size)
```

Comment:
Any particular reason for using rvalue reference here?

Reply:
oh nvm, it looks like all the drawing functions do that.

Reply:
Still seems bizarre